src/utils/index.ts

```diff
@@ -17,4 +17,6 @@ const showToast = async (message: string) => {
       return toast.present();
 }
 
+
+
```

Comment:
Remove extra lines that are not required.

Reply:
Done